components/lib/inputtext/InputText.spec.js

```diff
@@ -27,7 +27,6 @@ describe('InputText', () => {
 
         // Act
         expect(input).toBeEnabled();
-        expect(input).not.toHaveClass('p-filled');
```

Comment:
This is not needed, as the snapshot will cover this check.

Reply:
I left this here because it was explicitly what i was testing not meant to be implicitly figured out from the SNAPSHOT.